config/src/main/java/com/alibaba/nacos/config/server/service/sql/ModifyRequest.java

```diff
@@ -33,6 +33,8 @@ public class ModifyRequest implements Serializable {
     
     private String sql;
     
+    private boolean rollBackOnUpdateFail = Boolean.TRUE;
```

Comment:
default value should be “false” to compatible the former

Reply:
And only in config cas update method ,this value should be true,or you should test all scenes that affected by this modification